pkg/sidecar/sidecar.go

```diff
@@ -17,59 +17,96 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/andres-erbsen/clock"
+	proto "github.com/spiffe/go-spiffe/proto/spiffe/workload"
 	"github.com/spiffe/spire/api/workload"
```

Comment:
The spiffe-helper should start consuming the newer go-spiffe/workload package instead of depending on SPIRE but that is outside of the scope of this change.

Reply:
Thanks, Ill look into that after this is merged.

Reply:
Added a task in our internal jira for this.